src/views/Pools/components/PoolCard/CardActions/HarvestActions.tsx

```diff
@@ -48,35 +48,31 @@ const HarvestActions: React.FC<HarvestActionsProps> = ({
   return (
     <Flex justifyContent="space-between" alignItems="center" mb="16px">
       <Flex flexDirection="column">
```

Comment:
I wonder if this change impact the Flex style here?

Reply:
thanks, let me take a look